roles/keystone/tasks/openidc.yml

```diff
@@ -26,3 +26,45 @@
 - name: enable apache mod auth_openidc
   apache2_module: name=auth_openidc
   notify: reload apache
+
+- name: create metadata folder for multiple oidc configuration files
+  file:
+    path: "/etc/apache2/openidc/metadata/"
+    state: directory
+    owner: www-data
+    group: root
+    mode: 0775
+    recurse: yes
+
+- name: create .client file for the IdPs
+  template:
+    src: etc/apache2/openidc/metadata/template.client
+    dest: /etc/apache2/openidc/metadata/{{ item.issuer | regex_replace('https://','') }}.client
```

Comment:
doing the regex_replace seems a little weird... Can we just have a variable for this which has characters already safe for folder names?